accounts/identity.go

```diff
@@ -160,9 +159,5 @@ func parseKeystoreUrl(path string) (string, error) {
 // return default keystore directory stored in in `.sonm` directory
 // if any error occurred .sonm directory will be in working dir
 func GetDefaultKeystoreDir() string {
-	rootDir, err := util.GetUserHomeDir()
-	if err != nil {
-		rootDir = ""
-	}
-	return filepath.Join(rootDir, ".sonm", "keystore")
+	return filepath.Join("etc", "sonm", "keystore")
```

Comment:
I think it not a pretty good idea. Is there any reason to use `CWD/etc/sonm/keystore` dir as default key storage?